os_faults/drivers/tcpcloud.py

```diff
@@ -29,7 +29,7 @@ class TCPCloudNodeCollection(node_collection.NodeCollection):
 
     def connect(self, network_name):
         LOG.info("Connect network '%s' on nodes: %s", network_name, self)
-        task = {'osa_network_mgmt': {
+        task = {'fuel_network_mgmt': {
```

Comment:
Why the change from osa_netowrk_mgmt to fuel_network_mgmt?  I'm guessing there are some existing functions for fuel that "just work" and aren't really fuel-specific?